src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href=""
+        class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="container">
+        <a href="" class="container__element is-active">apple</a>
```

Comment:
```suggestion
        <a href="#" class="container__element is-active">apple</a>
```
Не оставляй href пустым